src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -200,6 +200,36 @@ void uct_ib_mlx5_get_av(struct ibv_ah *ah, struct mlx5_wqe_av *av)
     memcpy(av, &ucs_container_of(ah, struct mlx5_ah, ibv_ah)->av, sizeof(*av));
 }
 
+ucs_status_t uct_ib_mlx5_get_compact_av(uct_ib_iface_t *iface, int *compact_av)
+{
+    struct mlx5_wqe_av mlx5_av;
+    struct ibv_ah *ah;
+    uct_ib_address_t ib_addr;
+    ucs_status_t status;
+    int is_global;
+
+    status = uct_ib_iface_get_device_address(&iface->super.super,
```

Comment:
Found by coverity - please use ucs_alloca() to allocate the address of the correct size. see the iface query function. uct_ib_address_t is just the "header" of the address. this code results in stack corruption.